functions/task2.js

```diff
@@ -0,0 +1,6 @@
+function isAdult ( age ) {
```

Comment:
you can name it isAdult but I would consider rename the function so the would represent the action

Reply:
Updated the names of function.